ui/Gruntfile.js

```diff
@@ -36,6 +36,8 @@ module.exports = function (grunt) {
         'components/angular-recursion/angular-recursion.min.js',
         'components/ngInfiniteScroll/build/ng-infinite-scroll.min.js',
         'components/moment/min/moment-with-locales.min.js',
+        'components/moment/min/moment.min.js',
```

Comment:
Hi @sai789, 
I don't think you need `components/moment/min/moment.min.js` as `components/moment/min/moment-with-locales.min.js` will take care of all **moment** related stuff

Reply:
Done